src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -124,7 +128,10 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
         if p["filter"] and p["freq_min"] is not None and p["freq_max"] is not None:
             if verbose:
                 print("filtering")
-            recording = bandpass_filter(recording=recording, freq_min=p["freq_min"], freq_max=p["freq_max"])
+            # important to use dtype=float here
+            recording = bandpass_filter(
+                recording=recording, freq_min=p["freq_min"], freq_max=p["freq_max"], dtype=float
```

Comment:
you should use np.float32 to limit the memory footprint

Reply:
Ah thanks! I will fix.